### DISCUSSION TITLE
System Prompt

### DISCUSSION THREAD

**Discussion by alchemical-tao** [Vali-98/ChatterUI#142] [General]
Where is the best place to insert a custom system prompt that has such things as "be concise, use headings, follow up with 3 questions, etc"  This is an essential part of interfacing with LLMs, and I'm not seeing it in the app.  I see personality, scenario, and some other fields, and I've tried inserting system prompts into these, and I'm still unsure whether or not it is accepting these when I do testing.  If there is no defined field for a system prompt?  Should one just dump the prompt at the beginning of each message?  Is a system prompt field in the works?

Thanks

**Comment by Vali-98** [comment]
The formatter exists in Instruct screen, which includes the prefixes, suffixes, end of line and System Prompt.
